elastalert/config.py

```diff
@@ -251,7 +253,7 @@ def load_modules(rule, args=None):
 
     # Make sure we have required alert and type options
     reqs = rule['type'].required_options
-    for alert in rule['alert']:
+    for alert in rule_alerts:
         reqs = reqs.union(alert.required_options)
     if reqs - frozenset(rule.keys()):
```

Comment:
This looks like it will still throw an error if a required option is inline but not at the top level. I think the required_options checks have to be done per-rule now, probably after the copying happens.